### DISCUSSION TITLE
Teams-AI for dotnet

### DISCUSSION THREAD

**Discussion by YuriyMorozyuk95** [microsoft/hack-together-teams#8] [Q&A] [answered]
Do there is approximate date when Teams-AI will be avaliable for dotnet?
It possible to use Azure.AI.OpenAI SDK instead Teams-AI for AI based project?

**Comment by therealjohn** [comment, marked as answer]
The .NET bits are actively being developed, no timeline to share right now. 

You can use anything else you want - you'd just use the Bot Framework SDK for the bot pieces. You could call OpenAI APIs yourself too, or use the Azure OpenAI service.
